### PR TITLE
Add new CoCo to policies

### PR DESCRIPTION
**Pull request by mwcraig**

### CODE REVIEW

**Review by kelle — APPROVED** [review]

File: policies/coco-operating-policies.md [hunk 1]
```diff
@@ -96,6 +96,14 @@ CoCo members typically have owner privileges of the Astropy organization, but it
 
 ## CoCo membership history
 
+### CoCo4 (Election October 2022)
+
+- Kelle Cruz - term ends at 2024 election
+- Moritz Gunther - term ends at 2023 election
+- Derek Homeier - term ends at 2025 election
+- Pey Lian Lime - term ends at 2025 election
```
Comment (pllim) [inline policies/coco-operating-policies.md:104]:
```suggestion
- Pey Lian Lim - term ends at 2025 election
```

Comment (mwcraig) [reply policies/coco-operating-policies.md:104]: 😱 sorry about that!

Comment (pllim) [reply policies/coco-operating-policies.md]: np 😸

**Review by pllim — APPROVED** [review]

**Comment by pllim** [comment]
Thanks!

**Pull request merged**